views/management/accounts.php

```diff
@@ -59,8 +59,17 @@
                 return "<a href='/management/accounts/" . $user->id . "'  id='modify-account'>Modify Account<a/>";
             }
         ],
+        [
+            'attribute' => 'reset',
+            'label'     => 'Reset',
+            'format'    => 'raw',
+            'value'     => function ($user)
+            {
+                /** @var \app\models\databaseModels\User $user */
 
-
+                return "<a href='/management/reset-password/" . $user->id . "'  id='reset-password'>Reset User Password<a/>";
```

Comment:
CSRF : NEVER use a link (GET method) to EDIT the application state. Do a POST form